### PR TITLE
Comments regarding Overview section

### PR DESCRIPTION
**Pull request by InaOana**
I have updated the overview section with all the comments in the following .pdf document besides the ones marked in red (questions to Jeff) and in blue (TO DO for myself).
[ring_vrf_17jan.pdf](https://github.com/w3f/ring-vrf/files/10489955/ring_vrf_17jan.pdf)

### CODE REVIEW

**Comment by swasilyev** [comment]
[ring_vrf_17jan.pdf](https://github.com/w3f/ring-vrf/files/10489956/ring_vrf_17jan.pdf)

**Comment by InaOana** [comment]
All comments discussed in the meeting were updated in the overview.tex file.

File: papers/ring_vrf/overview.tex [hunk 1]
```diff
@@ -44,34 +44,34 @@ \section{Protocol overview}
...
-        & \Out = \Eval(\sk,\msg)
+        & \Out = \rVRF.\Eval(\sk,\msg)
         \end{aligned}
     }
 } $$
 % TODO:  \PRF vs \rVRF.\Eval here??
 % Although convenient for security arguments, % formalization
-Zero-knowledge here says our verifier learns nothing about the specific
-signer, except that their key is in the ring and maps $\msg \mapsto \Out$.
+The zero-knowledge property of the NIZK ensures that our verifier learns nothing about the specific
```
Comment (burdges) [inline papers/ring_vrf/overview.tex:66]: I might rephrase this again in future, but fine.

File: papers/ring_vrf/overview.tex [hunk 2]
```diff
@@ -92,28 +92,28 @@ \section{Protocol overview}
...
 makes \aux essential,%
 \footnote{If ring VRFs authorized creating blocks in an anonymous Praos blockchain then \aux must include the block being created, or else others could steal their block production turn.}
-but thankfully our ring VRFs expose \aux exactly like EC VRFs do in \S\ref{sec:pederson_vrf}.%
+but thankfully our ring VRFs in in \S\ref{sec:pederson_vrf} expose \aux exactly like EC VRFs should do .%
 \footnote{We suppress multiple input-output pairs until \S\ref{subsec:multi_io} below, but they work like in \cite{PrivacyPass} too.}
 
 % \smallskip
 
-We would need time $O(|\ring|)$ in \rSign and \rVerify merely to read
+Taking into account only the $\rVRF$ interface described above, we would need time $O(|\ring|)$ in \rVRF.\rSign and \rVRF.\rVerify merely to read
```
Comment (burdges) [inline papers/ring_vrf/overview.tex:100]: I might rephrase this again in future, but fine.

File: papers/ring_vrf/overview.tex [hunk 3]
```diff
@@ -34,8 +34,8 @@ \section{Protocol overview}
 
 % As an instructive but insecure over simplification, 
 At their simplest,
-ring VRFs' other algorithms might operate directly upon
- the plausible signer set \ring, like:
+ring VRFs' other algorithms operate directly upon
+ the alleged signer set \ring, like:
```
Comment (burdges) [inline papers/ring_vrf/overview.tex:38]: alleged has a different meaning that does not quite fit.

File: papers/ring_vrf/overview.tex [hunk 4]
```diff
@@ -127,30 +127,31 @@ \section{Protocol overview}
 
 % \smallskip
 
-After these corrections,  our algorithms for a
-{\em ring verifiable random function with associated data} (rVRF-AD)
-now really look like
+Having these concerns in mind, the full name our new notion is \emph{ring verifiable function with additional data}; 
+it has the same api as the \rVRF notion described above with the mention that signing and verifying are now defined as per below. Note that for simplicity and readability,
+we use only the prefix \rVRF:
```
Comment (burdges) [inline papers/ring_vrf/overview.tex:132]: I think this parses wrong as the having clause referring to our new notion, but our new notion does not have anything in mind.   I avoid a number of clause types in which I do not really know the correct rules off the top of my head, but of course some diversity is good too, and english is fairly forgiving about mildly incorrect usage.
https://www.sitepoint.com/community/t/start-sentence-with-having/208692/10
https://english.stackexchange.com/questions/375202/starting-a-sentence-with-having-or-when-provided-with

File: papers/ring_vrf/overview.tex [hunk 5]
```diff
@@ -175,28 +176,28 @@ \section{Protocol overview}
...
 } $$
 
-We discovered this becomes incredibly efficient if one specializes
-the Groth16 construction:  An inner true Groth16 for $\Lring^\inner$
+We discovered the SNARK for the language \Lring becomes incredibly efficient for the prover if one specializes
+the original Groth16 SNARK construction:  An inner original Groth16 SNARK for $\Lring^\inner$
 handles the secret key \sk directly via its public inputs, but
 \sk and even \pk remain secret by transforming the trusted setup to have
-a rerandomizable Pedersen commitment \compk outside this Groth16.
+a rerandomizable Pedersen commitment \compk outside this SNARK.
```
Comment (burdges) [inline papers/ring_vrf/overview.tex:183]: It does kinda matter that it's groth16 here imho

**Comment by burdges** [comment]
I'll maybe rephrase a couple things eventually, but looks fine.  Were there other things you wanted to include?

File: papers/ring_vrf/overview.tex (hunk 3 above)
Comment (InaOana) [reply papers/ring_vrf/overview.tex]: No problem. Reverted to plausible.

File: papers/ring_vrf/overview.tex (hunk 4 above)
Comment (InaOana) [reply papers/ring_vrf/overview.tex:132]: You are right, the sentence indeed starts with "Having", however it does not start with a past participle "Having been" which is explicitly mentioned in the first link. Both constructions can be problematic, I agree, but they are not identical.
In order to avoid "having", would the following work better: "Because of these concerns, the full name for our new notion ..."?

File: papers/ring_vrf/overview.tex (hunk 5 above)
Comment (InaOana) [reply papers/ring_vrf/overview.tex]: OK, I have updated to "this Groth16 SNARK".

**Comment by InaOana** [comment]
> I'll maybe rephrase a couple things eventually, but looks fine. Were there other things you wanted to include?

All the things I wanted to include in this PR today are here. Anything left over, will be included in the next PR(s).
BTW, I took into account all your comments above.
